tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -451,33 +428,38 @@ def test_positive_provision_rhev_with_host_group(rhev, provisioning, tear_down):
             'datacenter': rhev.datacenter,
             'url': rhev.hostname,
             'ovirt-quota': rhev.quota,
-            'organizations': provisioning.org_name,
-            'locations': provisioning.loc_name,
+            'organizations': org_name,
+            'locations': loc_name,
         }
     )
     assert rhv_cr['name'] == name
     host_name = gen_string('alpha').lower()
-    host = make_host(
+    host = cli.Host.create(
         {
             'name': f'{host_name}',
-            'root-password': gen_string('alpha'),
-            'organization': provisioning.org_name,
-            'location': provisioning.loc_name,
+            'root-password': rhev.root_password,
+            'organization': org_name,
+            'location': loc_name,
             'pxe-loader': 'PXELinux BIOS',
-            'hostgroup': provisioning.config_env['host_group'],
+            'hostgroup': hostgroup.name,
             'compute-resource-id': rhv_cr.get('id'),
             'compute-attributes': f"cluster={rhev.cluster_id},"
             "cores=1,"
-            "memory=1073741824,"
+            "memory=4294967296," # 4 MiB
             "start=1",
             'ip': None,
             'mac': None,
-            'interface': f"compute_name=nic1, compute_network={rhev.network_id}",
-            'volume': f"size_gb=10,storage_domain={rhev.storage_id},bootable=True",
+            'interface': f"compute_name=nic1, compute_network=SatQE-Network-{settings.provisioning.vlan_id}",
```

Comment:
The `SatQE-Network` portion of this shouldn't be hardcoded.

I'm also a bit suspicious of using a value from `settings.provisioning` for this, instead of a value from `settings.rhev`, since its referencing a compute network specific to the compute resource.

Reply:
Names of those networks are SatQE-Network-X for X being the VLAN that is used to create the provisioning Sat. It's directly dependent on what is in conf/provisioning, you can't have different values for provisioning Sat and for the RHEV VM.

Reply:
Understood - but it can't be hardcoded like this. It's hardcoding assumptions about source code and actions well outside of robottelo. 

The field in the yaml file for `settings.rhev.provisioning_network` could use dynaconf formatting to append `settings.provisioning.vlan_id`, or a base network name could be read from `settings.rhev.provisioning_network` and `settings.provisioning.vlan_id` appended here.  The second still assumes that RHEV has a network created with the VLAN ID as part of the name.

Reply:
Yes, the second option is what I'm planning to do.

Reply:
Thanks @lhellebr !